src/utility.js

```diff
@@ -8,3 +8,60 @@ export const fetchUrl = (url) =>
       return response.json();
     }
   });
+
+export async function getBanner(url) {
+  const fetchData = await fetchUrl(url);
+  if (fetchData !== undefined) {
+    const random_image = await fetchData.photos[
+      Math.floor(Math.random() * fetchData.photos.length)
+    ];
+    return random_image.src;
+  }
+  return {};
+}
```

Comment:
Try not to use `async...await` unless required. We don't want to convert every async operation to work like synchronous one